DXRCore/DeusEx/Classes/DXRActorsBase.uc

```diff
@@ -1734,3 +1734,13 @@ function DeusExGoal AddGoalFromConv(#var(PlayerPawn) player, name goaltag, name
 
     return goal;
 }
+
+static function bool ConsumableWouldHelp(DeusExPlayer player, Inventory item) {
+    return (
+        player.health < player.default.health &&
+        (MedKit(item) != None || SoyFood(item) != None || Candybar(item) != None || (HealingItem(item) != None && HealingItem(item).health > 0))
```

Comment:
HealingItem class doesn't exist in other mods, will cause compiler errors

but this whole function isn't needed in other mods, so you could #ifdef it out, or move it

Reply:
Moved to DXRPlayer